### ISSUE TITLE
Pre-pulled images causes an incorrect link in /var/run

### ISSUE THREAD

**Issue by uablrek** [Nordix/xcluster#60]
Some applications _requires_ that `/run` and `/var/run` exists and that they are the same directory.
```
~ > ls -l /var/run
lrwxrwxrwx 1 root root 4 Apr  8 08:10 /var/run -> /run/
```

In `xcluster` the link is setup, but is corrupted by the "pre-pulled" images in `ovl/images`. It creates a directory at "/var/run" and the link becomes:
```
vm-002 ~ # ls -l /var/run/run
lrwxrwxrwx    1 root     root             4 Sep 16 08:00 /var/run/run -> /run/
```

I have never been happy with the pre-pulled images (#8), so now may be a good time to ditch them? That should require a new release, and perhaps a new major version.

**Comment by uablrek** [comment]
If the major release is stepped, another major change may be to drop the `hd-k8s-xcluster.img` and have just one K8s image with capability to run a configurable CNI-plugin, but with a default (bridge).

**Comment by uablrek** [comment]
I have unified the k8s images in #61, and tested to drop the pre-pulled images. It works fine and the only problem is the `pause` image. It is pre-pulled but must now be in the local registry:
```
images lreg_cache registry.k8s.io/pause:3.9
```
This is already the case if `containerd` is used since pre-pulled images only work with `cri-o` in xcluster.

The pause image is rarely updated, but it should be synced with the version used in `kubeadm`:
```
cd $GOPATH/src/k8s.io/kubernetes
_output/bin/kubeadm config images list
registry.k8s.io/kube-apiserver:v1.28.2
registry.k8s.io/kube-controller-manager:v1.28.2
registry.k8s.io/kube-scheduler:v1.28.2
registry.k8s.io/kube-proxy:v1.28.2
registry.k8s.io/pause:3.9
registry.k8s.io/etcd:3.5.9-0
registry.k8s.io/coredns/coredns:v1.11.1
```

To drop pre-pulled images is a _very_ small update in `xcadmin.sh` but should include quite a lot of doc updates

**Comment by sriramy** [comment]
I don't fully understand pre-pulled images. I have to read the code a bit more, but I merged #61 now.

I love the idea of dropping hd-k8s-xcluster.img. That was my thinking when I wrote this comment: https://github.com/Nordix/xcluster/pull/50#issuecomment-1683590184. Although I was mistaken what k8s-cni-bridge did. :)

**Comment by uablrek** [comment]
Normally images are downloaded via the network, but for reasons like security or bad/no network connectivity, images can be "pre-pulled" before K8s starts. This is really simple if you do it from a node:
```
vm-004 ~ # crictl images
IMAGE               TAG                 IMAGE ID            SIZE
vm-004 ~ # crictl pull registry.k8s.io/pause:3.9
Image is up to date for registry.k8s.io/pause@sha256:8d4106c88ec0bd28001e34c975d65175d994072d65341f62a8ab0754b0fafe10
vm-004 ~ # crictl images
IMAGE                   TAG                 IMAGE ID            SIZE
registry.k8s.io/pause   3.9                 e6f1816883972       748kB
```
But to do it on the build host in an xcluster ovl requires "sudo" (for very silly reasons, like hard-coded paths), and is a messy process (or at least was in 2018).

I will create a PR with at least enough doc updates to remove obsolete instructions.

**Comment by sriramy** [comment]
Ok it is as simple as that😊. Thank you. There seems to have been a lot of updates to podman to run pods as non root. Perhaps skopeo also has improved and git support for creating non root images using container-storage.

**Comment by sriramy** [comment]
Also, tomorrow we will get linux-6.5.4 with the Antrea IPv6 distribution and TCP resets with ECMP problems fixed. So we could get that in for the new major version

**Comment by uablrek** [comment]
Fixed by #62
